### DISCUSSION TITLE
Zeno Rocha - React Email, Resend, Dracula

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#265] [Announcements]
- https://twitter.com/zenorocha
- https://react.email/
- https://resend.com/home
- https://draculatheme.com

### Dracula

- [x] Before your company the thing you were known most for is the Dracula Code theme. How did it start and how did you monetize it?

### react.email

- [x] What is react email? What about coding emails is hard?
- [x] How does it differ from MJML?
- [x] Was it always the plan to make a great OSS offering, then build a company on it?
- [x] Email is a pretty old medium at this point. It's amazing that it support any html at all, but we're in 2024. What are some features that you wish email had that the platform doesn't support?

### Resend

- [x] What is resend and why can't I just run react email by myself?
- [x] How'd y'all make the beautiful animations on the landing page?
- [x] What are some of the things that are hard to get right when sending this much email at scale?
- [x] On resend you support other platforms like java, python etc. Seemingly the only platform that supports react emails is next.js. Are there any plans to  support other platforms here?
- [x] You all recently wrapped up "resend forward" where you and your team added a bunch of features, could you give us some highlights?

### Future

- [x] What are the next big things you're working towards?
- [x] What do you think the future of email is?

**Comment by hipstersmoothie** [comment]
Tooltips

https://www.npmjs.com/package/workerpool
https://github.com/sindresorhus/promise-fun
https://react-postgres-components.vercel.app/

**Comment by hipstersmoothie** [comment]
[http://raycast.com](https://t.co/LNe1zel7XB) / [http://arc.net](https://t.co/dbePrCAgPY) / [https://walden.us/products/lava-rock-diffuser](https://t.co/Ft7c5kVdoj)

**Comment by just-be-dev** [comment]
Tooltips:

- https://github.com/esm-dev/esm.sh/releases/tag/v135
- https://www.actionforge.dev/
- https://sunilpai.dev/posts/the-future-of-serverless/
